ci/checks/run-all-checks.sh

```diff
@@ -24,6 +24,7 @@ enforce-newline-at-eof.sh
 enforce-release-notes-up-to-date.sh
 enforce-shell-scripts-pass-shellcheck.sh
 enforce-version-bumped.sh
+enforce-docs-up-to-date.sh
```

Comment:
Pls note that this script is supposed to be wired up as pre-commit hook (see CONTRIBUTING.md)... I'm not sure if running `pip install` on each commit is safe/reasonable (as the packages might end up getting installed in a venv pointed by the current shell etc.)

Reply:
Actually... it looks to me that this task better fits into `tox` model (with deps installed into a fresh virtual env) than into a vanilla shell script as now

Reply:
Yes, great idea, i think it should work well within tox